src/Components/Dictionary/WordTerm.css

```diff
@@ -0,0 +1,49 @@
+.Term {
+    font-family: 'Work Sans', sans-serif;
+    font-weight: 600;
+    font-size: 24px;
```

Comment:
The font sizes on the figma are not always accurate to every screen size. To make it look proportional, you might need to do some experimentation!